stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -1475,31 +1475,31 @@ func.func @clamp_compatible_dynamic_match_static(%arg0: tensor<?xi32>, %arg1: te
 
 // -----
 
-func.func @clamp_invalid_clamp_element_type(%arg0: tensor<1xi32>, %arg1: tensor<1xf32>) -> tensor<1xi32> {
+func.func @clamp_c3(%arg0: tensor<1xi32>, %arg1: tensor<1xf32>) -> tensor<1xi32> {
```

Comment:
For all `clamp` ops, can you sort them alphanumerically, including non-constraint tests? E.g. `@clamp_scalar` should come after all the constraint tests.

Reply:
Sure.